src/lib/dialog/OVERVIEW.md

```diff
@@ -37,3 +38,10 @@ Several directives are available to make it easier to structure your dialog cont
 | `<md-dialog-content>` | Primary scrollable content of the dialog                                 |
 | `<md-dialog-actions>` | Container for action buttons at the bottom of the dialog                 |
 | `md-dialog-close`     | \[Attr] Added to a `<button>`, makes the button close the dialog on click|
+
+Once a dialog opens, the dialog will automatically focus the first tabbable element.
+> To avoid that an element will be focused upon opening, developers can set the `tabindex`.
```

Comment:
This shouldn't be a quotation (using the `>` character). 

I would also phrase is as 
```
You can control which elements are tab stop with the `tabindex` attribute.
```